File: Array/WhereIsIt/WhereIsIt.java
//Array WhereIsIt 90pts Nico Liao
import java.util.*;
public class WhereIsIt {
    public static void main(String[] args) {
        Scanner keyboard = new Scanner(System.in);
		Random r = new Random();
		int arr[] = new int[10];
        int enter;
        int slot = 0;
        boolean found = false;
        System.out.print("Array: ");
		for (int i=0; i<arr.length; i++) {
			arr[i] = 1 + r.nextInt(99);
			System.out.print(arr[i] + " ");
		}
        System.out.print("Value to find: ");
        enter = keyboard.nextInt();
        for (int i=0; i<arr.length; i++) {
			if (arr[i] == enter) {
                slot = i;
                found = true;
            }
		}
        if (found) {
            System.out.println(enter + " is in slot " + slot + ".");

        }else {
            System.out.println(enter + " is not in the array.");

        }
    }
}

File: C++/Conversion3/WhereIsIt/WhereIsIt.cpp
//functions WhereIsIt 90pts Nico Liao
#include <iostream>
#include <string>
#include <stdlib.h>
#include <time.h> 
using namespace std;
int main(){
    srand (time(NULL));
    int value;
    int array [10];
    cout << "Array: ";
    for(int i=0; i<10; i++){
        array[i]=rand() % 50 + 1;
        cout<<array[i]<< " ";
    }
    cout << endl;
    cout << "Value To Find: ";
    cin >> value;
    for(int i=0; i<10; i++){
        if(array[i] == value) {
            cout << value << " is in slot " << i << endl;
        }else {
            cout << value << " is not in the array" << endl;
        }
    }
}